src/api/operations.ts

```diff
@@ -237,6 +237,15 @@ export const drawRectangle = (options: {
   matrix?: TransformationMatrix;
   clipSpaces?: Space[];
 }) => {
+  if (
```

Comment:
The logic is hard to understand written like that. Let's simplify:

```js
if (options.color || (options.borderWidth && options.borderColor)) {
  // Do stuff
} else {
  // No need to draw the rectangle
  return []
}
```

Alternatively, you could do:

```js
if (!options.color && (!options.borderWidth || !options.borderColor)) return []
```

Reply:
That is much cleaner, and your `getDrawingOperator` suggestion also makes a lot of sense. But I noticed that this logic is the same as the logic needed in `getDrawingOperator`, and I don't think it makes sense to have it _twice_, so I went with using the `getDrawingOperator` function.